packages/story-editor/src/components/library/panes/media/common/mediaElement.js

```diff
@@ -216,7 +191,7 @@ function Element({
       onFocus={makeActive}
       onPointerLeave={makeInactive}
       onBlur={makeInactive}
-      tabIndex={index === 0 ? 0 : -1}
+      tabIndex="-1"
```

Comment:
🤔  this shouldn't be needed at all, `Container` is a `div` which isn't focusable by default.  That said, if we ever wanted to do aXe testing here, this set up might make it angry because it has `onFocus`, etc. So, should this be a button?

Reply:
Thanks so much for reviewing the PR, Brittany! It might need a bit too much refactoring for this PR to change it to a button since it currently already contains a button (the dropdown menu) + has to also work with the Moveable for dragging etc. I will create a follow-up issue for looking into changing these to buttons (here and elsewhere where these are currently not buttons).

Reply:
totally agree it's a good follow up - not anything for right now.